src/main/webapp/WEB-INF/views/car/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create car</title>
+</head>
+<body>
+    <h1> Create car</h1>
+    <form method="post" name="${pageContext.request.contextPath}/car/add">
+        Model <input type="text" name="model">
```

Comment:
try to create a new car through this UI and then check your db and logs,
can you create a car or manufacturer, or driver?